### PR TITLE
🔹

### PR DESCRIPTION
**Pull request by elianiva**
[青](https://music.youtube.com/watch?v=dGZqpVCJP3k&list=RDAMVM6foIBJdSzbo)

### CODE REVIEW

File: backend/src/constant/options.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+export type Option = {
+  type: 'string' | 'number' | 'boolean' | 'keys';
+  isRequired: boolean;
+  min?: number;
+  max?: number;
+  options?: string[];
+  errorMessage?: string;
+};
+
+type Options = Record<string, Options | Option>;
```
Comment (aldy505) [inline backend/src/constant/options.ts:12]: Just define the schema as it is. No need to do this.

**Review by aldy505 — COMMENTED** [review]
can you change eslint to

@teknologi-umum/rslint-config-typescript? thanks

**Comment by elianiva** [comment]
eslint di PR terpisah aja, diffnya bakal gede

**Comment by aldy505** [comment]
ok

File: backend/src/logic/validate.ts [hunk 2]
```diff
@@ -1,71 +1,30 @@
-import { BUNDLED_LANGUAGES, BUNDLED_THEMES } from 'shiki';
+import { Option, VALID_OPTIONS } from '../constant/options';
```
Comment (aldy505) [inline backend/src/logic/validate.ts:1]: shouldnt this be `import type Option` ?

**Review by aldy505 — APPROVED** [review]

**Pull request merged**